sender.go

```diff
@@ -297,12 +296,44 @@ Loop:
 			outgoingTransfers = s.transfers
 		}
 
+		if len(outgoingDisps) > 0 && len(outgoingDisp) == 0 {
```

Comment:
It's really a shame that channels have to be bounded. :|

Reply:
I know right, an unbounded channel would really be handy for this specific case.